client/src/app/management/export/committee-csv-export-example.ts

```diff
@@ -0,0 +1,20 @@
+import { CommitteeCsvPort } from '../../shared/models/event-management/committee.constants';
+
+export const COMMITTEE_CSV_EXPORT_EXAMPLE: CommitteeCsvPort[] = [
+    {
+        name: 'Audit committee',
+        forward_to_committee_ids: 'Election committee',
+        organization_tag_ids: 'Audit'
+    },
+    {
+        name: 'Election committee',
+        description: 'This committee has to overwatch and perform elections',
+        forward_to_committee_ids: 'Finance committee',
+        organization_tag_ids: 'Election'
+    },
+    {
+        name: 'Finance committee',
+        description: 'A committee to observe financial activities',
+        organization_tag_ids: 'Financial'
+    }
+];
```

Comment:
every string has to be marked for translation. Also, during export the translated values need to be exported, so we can provide translated samples for exporting.